packages/app/package.json

```diff
@@ -23,10 +24,10 @@
     "@fuel-ts/mnemonic": "0.29.1",
     "@fuel-ts/wallet": "0.29.1",
     "@fuel-ts/wallet-manager": "0.29.1",
-    "@fuel-ui/config": "^0.10.1",
-    "@fuel-ui/css": "^0.10.1",
-    "@fuel-ui/react": "^0.10.1",
-    "@fuel-ui/test-utils": "^0.10.1",
+    "@fuel-ui/config": "^0.10.2",
+    "@fuel-ui/css": "^0.10.2",
+    "@fuel-ui/react": "^0.10.2",
+    "@fuel-ui/test-utils": "^0.10.2",
```

Comment:
this dependency change were made because of the linking recursive, right?

the projects don't really need that. can we revert?